src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -41,21 +42,21 @@ internal static void AssertOnlyValidationMessage(this IEnumerable<ValidationMess
     [Collection("Validation Tests")]
     public partial class SwaggerModelerValidationTests
     {
-        private IEnumerable<ValidationMessage> ValidateSwagger(string input)
+        private IEnumerable<ValidationMessage> ValidateSwagger(string input, ServiceDefinitionDocumentType serviceDefDocType = ServiceDefinitionDocumentType.ARM, 
```

Comment:
could you add comments to this? I think "mergeState" is not providing much information on what it represents, or how before or after will change the output of the test, if there's something that can help make the code more readable / easier understanding of the intent, I think it'd be good to do it.